### ISSUE TITLE
Can't install in sylius 1.9

### ISSUE THREAD

**Issue by jorgerucoba** [webgriffe/SyliusAkeneoPlugin#54]
Hello, I upgraded my version of Sylius to 1.9 and when I tried to install the SyliusAkeneoPlugin by `composer require webgriffe/SyliusAkeneoPlugin` , the following Exception is thrown:

```
Executing script cache:clear [KO]
 [KO]
Script cache:clear returned with error code 255
!!  PHP Fatal error:  Access level to Webgriffe\SyliusAkeneoPlugin\Command\QueueCleanupCommand::SUCCESS must be public (as in class Symfony\Component\Console\Command\Command) in /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/webgriffe/sylius-akeneo-plugin/src/Command/QueueCleanupCommand.php on line 101
!!  PHP Stack trace:
!!  PHP   1. {main}() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/bin/console:0
!!  PHP   2. Symfony\Bundle\FrameworkBundle\Console\Application->run() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/bin/console:38
!!  PHP   3. Symfony\Bundle\FrameworkBundle\Console\Application->doRun() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/console/Application.php:166
!!  PHP   4. Symfony\Bundle\FrameworkBundle\Console\Application->registerCommands() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/framework-bundle/Console/Application.php:74
!!  PHP   5. App\Kernel->boot() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/framework-bundle/Console/Application.php:168
!!  PHP   6. App\Kernel->preBoot() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/http-kernel/Kernel.php:121
!!  PHP   7. App\Kernel->initializeContainer() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/http-kernel/Kernel.php:780
!!  PHP   8. Symfony\Component\DependencyInjection\ContainerBuilder->compile() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/http-kernel/Kernel.php:541
!!  PHP   9. Symfony\Component\DependencyInjection\Compiler\Compiler->compile() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/dependency-injection/ContainerBuilder.php:736
!!  PHP  10. Symfony\Component\DependencyInjection\Compiler\AutowirePass->process() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/dependency-injection/Compiler/Compiler.php:91
!!  PHP  11. Symfony\Component\DependencyInjection\Compiler\AutowirePass->process() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:54
!!  PHP  12. Symfony\Component\DependencyInjection\Compiler\AutowirePass->processValue() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/dependency-injection/Compiler/AbstractRecursivePass.php:46
!!  PHP  13. Symfony\Component\DependencyInjection\Compiler\AutowirePass->doProcessValue() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:72
!!  PHP  14. Symfony\Component\DependencyInjection\Compiler\AutowirePass->processValue() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:103
!!  PHP  15. Symfony\Component\DependencyInjection\Compiler\AutowirePass->processValue() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/dependency-injection/Compiler/AbstractRecursivePass.php:81
!!  PHP  16. Symfony\Component\DependencyInjection\Compiler\AutowirePass->doProcessValue() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:72
!!  PHP  17. Symfony\Component\DependencyInjection\Compiler\AutowirePass->autowireCalls() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:126
!!  PHP  18. Symfony\Component\DependencyInjection\Compiler\AutowirePass->autowireMethod() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:171
!!  PHP  19. Symfony\Component\DependencyInjection\Compiler\AutowirePass->Symfony\Component\DependencyInjection\Compiler\{closure:/Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:229-241}() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:261
!!  PHP  20. Symfony\Component\DependencyInjection\Exception\AutowiringFailedException->__construct() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:236
!!  PHP  21. Symfony\Component\DependencyInjection\Compiler\AutowirePass->Symfony\Component\DependencyInjection\Compiler\{closure:/Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:387-389}() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/dependency-injection/Exception/AutowiringFailedException.php:29
!!  PHP  22. Symfony\Component\DependencyInjection\Compiler\AutowirePass->createTypeNotFoundMessage() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:388
!!  PHP  23. Symfony\Component\DependencyInjection\Compiler\AutowirePass->createTypeAlternatives() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:407
!!  PHP  24. Symfony\Component\DependencyInjection\Compiler\AutowirePass->populateAvailableTypes() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:433
!!  PHP  25. Symfony\Component\DependencyInjection\Compiler\AutowirePass->populateAvailableType() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:323
!!  PHP  26. Symfony\Component\DependencyInjection\ContainerBuilder->getReflectionClass() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:337
!!  PHP  27. Symfony\Component\Config\Resource\ClassExistenceResource->isFresh() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/dependency-injection/ContainerBuilder.php:348
!!  PHP  28. class_exists() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/config/Resource/ClassExistenceResource.php:84
!!  PHP  29. spl_autoload_call() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/config/Resource/ClassExistenceResource.php:84
!!  PHP  30. Composer\Autoload\ClassLoader->loadClass() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/symfony/config/Resource/ClassExistenceResource.php:84
!!  PHP  31. Composer\Autoload\includeFile() /Users/jorge/Coding/Bitbucket/babyq-shop-sylius/vendor/composer/ClassLoader.php:322
```

Is it absolutely not possible to install it with that version of Sylius? Or is there any dependencies conflict?

Thanks in advance

**Comment by mmenozzi** [comment]
Hi, we started to work for the 1.9 compatibility on `master` but we're not ready yet.
I'll keep this issue open until we're ready to tag a 1.9 compatible version.

**Comment by mmenozzi** [comment]
Just released version [1.8.0](https://github.com/webgriffe/SyliusAkeneoPlugin/releases/tag/1.8.0) which adds Sylius 1.9 compatibility.
